### DISCUSSION TITLE
Avoid sending out invites as `noreply-<random>@domain.com`

### DISCUSSION THREAD

**Discussion by nebulade** [outline/outline#7949] [Self Hosting] [answered]
This came up in the Cloudron forum from a user at https://forum.cloudron.io/topic/12796/magic-signin-link-not-sent-since-1-7-0-0-81-0-update/7

After debugging this a bit, we found that user invites are sent out from a dynamic mail address. This breaks user-invite on systems where the SMTP credentials are not valid for random adresses.

Is there a specific reason why only invite emails are using this pattern at https://github.com/outline/outline/blob/bb43c24efef817cdbc76d4e90781355897280c0b/server/emails/templates/BaseEmail.tsx#L191 ?

When patching this to use `env.SMTP_FROM_EMAIL` like elsewhere the app works well of course.

I am happy to create a PR for this if this has any chance to get merged.

Thanks!

**Comment by tommoor** [comment, marked as answer]
This was reported and closed here https://github.com/outline/outline/issues/7929 – I'll release a patch build before the end of the week.

The reasoning for this change is to prevent a particular type of attack that depends on knowledge of the address emails that magic sign-in emails are being sent from, however it really makes more sense for the cloud-hosted version than self-hosted, so have just gone ahead and reverted that behavior in self-hosted.

**Reply by nebulade** [reply]
Ah great and thanks for the explanation and sorry that I didn't find that related issue myself :/
